### PR TITLE
Update TLS generate-certs default destination location

### PR DESCRIPTION
**Pull request by 5abeel**
A bug was raised where generates-certs.sh script was present in `/usr/share/stratum/es2k` folder and executing the script results in certs in `/usr/share/stratum/es2k/certs` location, while the default location that `infrap4d` uses is `/usr/share/stratum/certs`.

Updating the use expected default location + documentation change to reflect the changes.

### CODE REVIEW

File: docs/guides/security/using-tls-certificates.md
Comment (ffoulkes) [inline docs/guides/security/using-tls-certificates.md:1]: My two cents:
`/usr/share/stratum` is the parent directory. The certificate files belong in `/usr/share/stratum/certs`.

File: scripts/common/copy_config_files.sh.in [hunk 1]
```diff
@@ -30,15 +30,16 @@ sudo mkdir -p /var/log/stratum/
...
 sudo cp ${SOURCE_DIR}/@TARGET_NAME@_port_config.pb.txt ${TARGET_DIR}/
 sudo cp ${SOURCE_DIR}/@TARGET_NAME@_skip_p4.conf ${TARGET_DIR}/
 
 #... Install files required for TLS certificate generation ...#
-sudo cp ${SOURCE_DIR}/ca.conf ${TARGET_DIR}/
-sudo cp ${SOURCE_DIR}/grpc-client.conf ${TARGET_DIR}/
-sudo cp ${P4CP_INSTALL}/sbin/generate-certs.sh ${TARGET_DIR}/
+sudo cp ${SOURCE_DIR}/ca.conf ${TLS_CERTS_DIR}/
+sudo cp ${SOURCE_DIR}/grpc-client.conf ${TLS_CERTS_DIR}/
+sudo cp ${P4CP_INSTALL}/sbin/generate-certs.sh ${TLS_CERTS_DIR}/
```
Comment (ffoulkes) [inline scripts/common/copy_config_files.sh.in:42]: Why are we copying `generate-certs.sh` to `/usr/share/stratum`?

It's an administrative script. It belongs in the `sbin` directory.

**Review by ffoulkes — APPROVED** [review]
LGTM

File: scripts/common/copy_config_files.sh.in (hunk 1 above)
Comment (5abeel) [reply scripts/common/copy_config_files.sh.in:42]: Resolved after offline discussion with Derek

**Review by satish153 — APPROVED** [review]

**Review by nupuruttarwar — APPROVED** [review]
LGTM

**Comment by 5abeel** [comment]
Thanks for approval. 
Will wait until 1.0 is completed before merging this -- since the IPU SDK also needs changes and changes communicated with broader team

**Pull request merged**